stories/pagination/pagination.stories.js

```diff
@@ -18,30 +18,60 @@ Element | Modifier/class | Description
-Previous arrow | \`fd-pagination__link--previous\` | The previous arrow that users can use to navigate backward. This arrow is disabled when on the first page.
-Next arrow | \`fd-pagination__link--next\` | The next arrow that users can use to navigate forward. This arrow is disabled when on the last page.
+Total page count | \`fd-pagination__total\` | The area where total pages information is placed
+Total page count label | \`fd-pagination__total-label\` | The total number of pages label
+Links | \`fd-pagination__link\` | The page number links that users can select to navigate to a different pages
+Selected page | \`fd-pagination__input\` | The input with the page that is currently selected, can be used to navigate to the specific page 
+Selected page label | \`fd-pagination__input-label\` | The label of the selected page input. Shown only on mobile.
+First page button | \`fd-pagination__button--mobile\` | The first page button that users can use to navigate to the first page. This button is disabled when on the first page. Shown only on mobile.
+Previous page button | \`fd-pagination__button\` | The previous page button that users can use to navigate backward. This button is disabled when on the first page.
+Next page button | \`fd-pagination__button\` | The next arrow that users can use to navigate forward. This arrow is disabled when on the last page.
+Last page button | \`fd-pagination__button--mobile\` | The last page button that users can use to navigate to the last page. This button is disabled when on the last page. Shown only on mobile.
+Per page | \`fd-pagination__per-page\` | The area where items per page select & its label are placed
+Per page label | \`fd-pagination__per-page-label\` | Per page section label. Hidden on mobile.
 <br>
 <br>
 
       `,
         tags: ['a11y', 'theme'],
-        components: ['pagination']
+        components: ['select', 'button', 'icon', 'form-label', 'input', 'pagination', 'popover', 'list']
     }
 };
 
-export const firstPage = () => `<div class="fd-pagination">
-  <span class="fd-pagination__total">30 items</span>
-  <nav class="fd-pagination__nav">
-      <a href="#" class="fd-pagination__link fd-pagination__link--previous" aria-label="Previous"aria-disabled="true"></a>
-      <a href="#" class="fd-pagination__link is-selected" >1</a>
-      <a href="#" class="fd-pagination__link">2</a>
-      <a href="#" class="fd-pagination__link">3</a>
-      <a href="#" class="fd-pagination__link fd-pagination__link--next" aria-label="Next"
-      aria-disabled="false"></a>
-  </nav>
+export const firstPage = () => `<style>*[dir='rtl'] .fd-pagination__button { transform: rotate(180deg) !important; }</style>
+<div class='fd-pagination'>
+    <nav class='fd-pagination__nav'>
+        <div style=''>
+        
+        </div>
+        <a href='#' class='fd-button fd-button--compact fd-button--transparent fd-pagination__button fd-pagination__button--mobile' aria-label='First' aria-disabled='true'>
```

Comment:
why do you have the `fd-pagination__button--mobile` modifier in desktop example?

Reply:
Actually, there is no separate mobile navigation, when the screen's size is `< 600px` we will have first & last page buttons.